styx-common/src/main/java/com/spotify/styx/model/TriggerParameters.java

```diff
@@ -0,0 +1,38 @@
+/*-
+ * -\-\-
+ * Spotify Styx Common
+ * --
+ * Copyright (C) 2016 - 2018 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.styx.model;
+
+import io.norberg.automatter.AutoMatter;
+import java.util.Map;
+
+@AutoMatter
+public interface TriggerParameters {
```

Comment:
👍 
I see this will serve a good foundation for another feature we have been thinking of: user being able to provide `docker_args` when triggering in order to overwrite default ones and provide additional ones.

Reply:
⭐️